packages/design-system-scripts/gulp/build.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * @file The build tasks handle compiling and optimizing both the
+ *  package assets and the documentation site. Essentially makes
+ *  everything production-ready.
+ */
+const babel = require('gulp-babel');
+const count = require('gulp-count');
+const del = require('del');
+const gulp = require('gulp');
+const getPackageName = require('./common/getPackageName');
+const streamPromise = require('./common/streamPromise');
+const { compileSass } = require('./sass');
+const { printStats } = require('./stats');
+const { log, logTask, logIntroduction } = require('./common/logUtil');
+
+const CORE_PACKAGE_NAME = '@cmsgov/design-system-core';
```

Comment:
Is this temporary? Like after we publish`@cmsgov/design-system@4.0.0` this variable will change?

Reply:
Yeah, and that was my most recent source of error 😂😭